src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -859,7 +859,8 @@ printUsage(void)
  "               Java Debugger JDWP Agent Library\n"
  "               --------------------------------\n"
  "\n"
- "  (see http://java.sun.com/products/jpda for more information)\n"
+ "  (See the \"Oracle VM Invocation Options\" section of the JPDA\n"
```

Comment:
I think you can reduce to "Invocation Options" to avoid having "Oracle VM" in the usage output.